src/scenarios/crossgen2/test.py

```diff
@@ -0,0 +1,16 @@
+'''
+Crossgen2
+'''
+from shared.runner import Runner
+from shared.testtraits import TestTraits, testtypes
+from shared import const
+
+EXENAME = 'crossgen2'
+
+if __name__ == "__main__":
+    traits = TestTraits(exename=EXENAME,
+                        startupmetric='Crossgen2',
+                        guiapp='false',
+                        timeout=const.MINUTE*30
```

Comment:
crossgen2 composite scenario takes longer time in general, depending on how many assemblies it compiles. It will take longer for PR builds on virtual machines. It is hard to measure exactly how long it will take but 30 min should be a safe number. We will see how long it takes once it runs on a regular basis and shorten the time if needed.

Reply:
crossgen2 composite scenario takes longer time in general, depending on how many assemblies it compiles. It will take longer for PR builds on virtual machines. It is hard to measure exactly how long it will take but 30 min should be a safe number. We will see how long it takes once it runs on a regular basis and shorten the time if needed.